.github/workflows/pull-request.yml

```
@@ -295,7 +295,116 @@
+            node_modules
+          key: ${{ runner.os }}-docker-check-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-docker-check-
+
+      - name: Check if Talawa API starts in Docker
+        run : | 
+          # Ensure no containers are running 
+          docker-compose -f docker-compose.dev.yaml down -v || true
+
+          # Verify docker-compose file exists
+          if [! -f "docker-compose.dev.yaml"]; then 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)